Add UKCC 7 + UKCC 8 tournament wiki pages (#3266)
ppy/osu-wiki: wiki/Tournaments/UKCC/UKCC7/en.md

---
tags:
- UK Community Cup 7
- UKCC 7
- UKCC 7
---

# UK Community Cup 7

![UKCC 7 logo](img/banner.png)

The seventh edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.

---

## Introduction

![Introduction](img/intro.png)

- This is a **1v1, UK and Ireland based tournament**
- In the event there is 128 signups, those with the lowest global rank will be unable to participate (unless we expand the tournament)
- The structure of the tournament will be:

> 	- Two qualifications rounds - 32 players progress

Author: This is an intended indentation - it's a sub list as shown
<img width="552" alt="Screenshot 2020-03-05 at 14 42 24" src="https://user-images.githubusercontent.com/31848108/75992211-8f254300-5eef-11ea-8092-2897911b6c8f.png">

### Warmups

- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less

### Picking

- Players have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
- For Freemod picks, players must select any combination of the following mods: **Hidden, HardRock**
- **NEW** - Failed scores WILL count
- Tiebreakers are Freemod, however No Mod can be used.

## Tournament schedule

![Schedules](img/schedules.png)

| Event | Timestamp |
| --: | :--|
| Signups Open | 2019-07-27 |
| Signups Closed | 2019-08-26 |
| Qualifying Round 1 | 2019-09-07/2019-09-08 |
| Qualifying Round 2 | 2019-09-14/2019-09-15 |
| Round of 32 | 2019-09-21/2019-09-22 |
| Round of 16 + Losers Round 1 | 2019-09-28/2019-09-29 |
| Quarter Finals + Losers Round 2/3 | 2019-10-5/2019-10-6 |
| Semi Finals + Losers Round 4/5 | 2019-10-12/2019-10-13 |
| Winners Finals + Losers 6/Semi Final | 2019-10-19/2019-10-20 |
| Losers Final + GRAND Finals | 2019-10-26/2019-10-27 |

## Prizes

![Prizes](img/prizes.png)

| Placing | Prize(s)|
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |
```
Missed 😉

| ![Silver Crown](/wiki/shared/crown-silver.png "Second place") | 3 months of osu! supporter and a sparkling medal |
| ![Bronze Crown](/wiki/shared/crown-bronze.png "Third place") | 1 month of osu supporter and a sparkling medal |

### Badge
![UKCC 7 Winner](img/badge.png)

## Organisation

![Staff List](img/staff.png)

> UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [Discord](https://discord.gg/EtsU7Cw)!

Reviewer: ```suggestion
UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [UKCC Discord server](https://discord.gg/EtsU7Cw)!
```

| 97-111 | ![][flag_GB] [Damarsh](https://osu.ppy.sh/users/7465147), ![][flag_GB] [DrexiN](https://osu.ppy.sh/users/5363646), ![][flag_GB] [cfman29](https://osu.ppy.sh/users/9023597), ![][flag_GB] [God](https://osu.ppy.sh/users/10502908), ![][flag_GB] [Mantle](https://osu.ppy.sh/users/7225966), ![][flag_GB] [aroS](https://osu.ppy.sh/users/7193474), ![][flag_IE] [Neverslain](https://osu.ppy.sh/users/10644912), ![][flag_IE] [kaniciwah](https://osu.ppy.sh/users/9371954), ![][flag_GB] [josph12](https://osu.ppy.sh/users/8584840), ![][flag_GB] [mamoopy](https://osu.ppy.sh/users/9384951), ![][flag_GB] [fadentic](https://osu.ppy.sh/users/13453905), ![][flag_GB] [Metzo](https://osu.ppy.sh/users/10633982), ![][flag_GB] [Bowris](https://osu.ppy.sh/users/8562850), ![][flag_GB] [_Phazy](https://osu.ppy.sh/users/3956707), ![][flag_IE] [Broznix](https://osu.ppy.sh/users/13909259) |

---

## Podium

UKCC 7 concluded on the 27th October 2019.

| Placing | Player |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "Winner") | [Bubbleman](https://osu.ppy.sh/users/5182050) |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) |
```
Nationality

| ![Silver crown](/wiki/shared/crown-silver.png "Second place") | [Spare](https://osu.ppy.sh/users/2204373) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "Third place") | [AJT](https://osu.ppy.sh/users/3181083) |

---

## Map pools

![Map pools](img/mappool.png)

*NB - Map packs may no longer be available due to age.*

### Losers Final + Grand Final

**[Download the map pack here! (148.0MB)](https://mega.nz/#!TmhgUK5K!7kkYZPdolGvJbuCezRBj1SFJe6J-qZRu0B4qXTwXwzU)**

> - Nomod

Reviewer: Indent following parent. (i think it's 2 instead of 4 or just follow the parent)

- Hidden
    - [xi - FREEDOM DiVE [Another]](https://osu.ppy.sh/beatmaps/126645)
    - [Yunomi & nicamoq - Indoor Kei Nara Trackmaker [Another]](https://osu.ppy.sh/beatmaps/1115999)
    - [DJKurara - Goodbye [SEVEN COLORS]](https://osu.ppy.sh/beatmaps/1811477)
- Hard Rock
    - [TAG - PRANA [Extreme]](https://osu.ppy.sh/beatmaps/234444)
    - [Yumeno Yuki - THE APPLE IS CAST! [Resolve]](https://osu.ppy.sh/beatmaps/1974604)
    - [Powerless - Soleil [#1: handsome & pishifat]](https://osu.ppy.sh/beatmaps/1523514)
- Double Time
    - [Toyosaki Aki - MORE&MORE [Insane]](https://osu.ppy.sh/beatmaps/318975)
    - [BRZion - Game Brain [Insane]](https://osu.ppy.sh/beatmaps/105978)
    - [kors k - Dot [korzhik]](https://osu.ppy.sh/beatmaps/441616)
    - [Hardgore Alice (CV: Hidaka Rina) - Forget me Not... [Insane]](https://osu.ppy.sh/beatmaps/1924965)
- Free Mod
    - [DECO*27 - Ghost Rule [z1085684963's Extra]](https://osu.ppy.sh/beatmaps/951521)
    - [DragonForce - Ashes of the Dawn [Extra]](https://osu.ppy.sh/beatmaps/1650328)
    - [CustomiZ - COOLEST [Hi-Speed Extreme]](https://osu.ppy.sh/beatmaps/1105536)
- Tiebreaker

>     - [Ata - Euphoria [Delusion]](https://osu.ppy.sh/beatmaps/1705819)

Reviewer: ```suggestion
    - **[Ata - Euphoria [Delusion]](https://osu.ppy.sh/beatmaps/1705819)**
```
Bold every Tiebreaker

    - [iru1919 - Tenko [Insane]](https://osu.ppy.sh/beatmaps/1759876)
- Double Time
    - [u's - START:DASH!! [Insane]](https://osu.ppy.sh/beatmaps/264252)
    - [Hana - Gekka Jasmine (2014ver.) [Gyro's Light Insane]](https://osu.ppy.sh/beatmaps/1035068)

*Maps were played in the order: NM3, HR2, DT2, HD2, HR1, NM2, DT1, NM1, HD1*

### Qualifying Round 1

**[Download the map pack here! (46.7MB)](https://mega.nz/#!T6RjDY7S!0PbquOystkS4v2-byiBC0b_KMEZA8lECFhMqjYKDm_A)**

- Nomod
    - [ALiCE'S EMOTiON - Cold Rain [Lunatic]](https://osu.ppy.sh/beatmaps/482650)
    - [sakuzyo - VALLISTA [Another]](https://osu.ppy.sh/beatmaps/127313)
    - [U - Ha-tenya? [Insane]](https://osu.ppy.sh/beatmaps/120080)
    - [YUC'e - Sengoku HOP [apple's Insane]](https://osu.ppy.sh/beatmaps/1489274)
- Hidden
    - [Iguchi Yuka - RE-ILLUSION [Collab Insane]](https://osu.ppy.sh/beatmaps/1856697)
- Hard Rock
    - [Meramipop - algorhythm [Insane]](https://osu.ppy.sh/beatmaps/136571)
- Double Time
    - [T.M.Revolution x Mizuki Nana - Preserved Roses [Collab Hard]](https://osu.ppy.sh/beatmaps/522135)

## Schedules / Match Results

![Schedules](img/schedule_no_number.png)

### Losers Final + Grand Final

> | Wednesday, 2019-10-23 | | | | |

Reviewer: We had a new format recently. Please follow example below and apply for every table in match results.
```
Wednesday, 23 October 2019

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Bubbleman** ![][flag_GB] | **7** | 4 | ![][flag_GB] AJT | [#1](link) |
```
Also the player 2's structure is flag first then the username

| **Harpy** ![][flag_GB] | **5** | 0 | Defex ![][flag_GB] | [#1](https://osu.ppy.sh/community/matches/55042213) |
| **helix** ![][flag_GB] | **5** | 1 | Rinnu ![][flag_GB] | [#1](https://osu.ppy.sh/community/matches/55043513) |
| **Apo11o** ![][flag_GB] | 1 | - | DeltaZero ![][flag_GB] | FF |

| Monday, 2019-9-23 | | | | |
| --: | :-: | :-: | :-- | :-- |
| -spoon- ![][flag_GB] | 4 | **5** | Wario ![][flag_GB] | [#1](https://osu.ppy.sh/community/matches/55061725) |

| Wednesday, 2019-9-25 | | | | |
| --: | :-: | :-: | :-- | :-- |
| **Ventus** ![][flag_GB] | **5** | 4 | Summer Solstice ![][flag_GB] | [#1](https://osu.ppy.sh/community/matches/55103447) |

### Qualifier Round 2

Full results: [LINK](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit#gid=1213327055)  
![Qualifier 2 Results table](https://i.imgur.com/XlM1Pr9.png)

### Qualifier Round 1

Full results: [LINK](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit#gid=1010206758)  
![Qualifier 1 Results table](https://i.imgur.com/pOTBzTC.png)

---

Cya next time 👋

*Written by ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972)*

<!-- Assets -->

> [flag_BE]: /wiki/shared/flag/BE.gif

Reviewer: ```suggestion
[flag_BE]: /wiki/shared/flag/BE.gif "Belgium"
```
Name every country.

---
tags:
- UK Community Cup 7
- UKCC 7
- UKCC 7
---

# UK Community Cup 7

> ![UKCC 7 Logo](img/banner.png)

Reviewer: ```suggestion
![UKCC 7 logo](img/banner.png)
```

> The 7th edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.

Reviewer: ```suggestion
The seventh edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.
```
hosted by? and which game mode was featured?

> # Contents

Reviewer: Don't think this whole contents will be very useful, since the site itself has a built-in table of contents in the sidebar.

Author: Good point - i'll take it out

[Introduction](#introduction)  
[Links](#links)  
[Rules](#rules)  
[Schedule](#tournament-schedule)  
[Prizes](#prizes)  
[Organisation](#organisation)  
[Players](#players)  
[Results](#results)  
[Map pools](#map-pools)  
[Schedules / Match Results](#Schedules-/-Match-Results)

---

## Introduction

![Introduction](img/intro.png)

- This is a **1v1, UK and Ireland based tournament**
- In the event there is 128 signups, those with the lowest global rank will be unable to participate (unless we expand the tournament)
- The structure of the tournament will be:

>     - Two qualifications rounds - 32 players progress

Reviewer: ```suggestion
    - Two qualifications rounds - 32 players progress
```
Line it up right below the first word of previous line. (Use tab)
Do for others too.

Author: I do use tab - but my editor is set to 4 space tab. Updated the doc to be 2 space.

- In order to reschedule, you must contact your opponents, agree on a time and be able to provide **PROOF** (by form of a screenshot/image) of the agreed reschedule to the Hosts.
- In effort to ensure as many matches take place as possible, reschedules are allowed up to **1 HOUR BEFORE YOUR MATCH IS DUE TO BE PLAYED**. However, your opponent holds the right to **refuse a reschedule if asked less than 24 hours from the scheduled time**
- If it is impossible to find a time to reschedule to that suits both players, both players should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
- Matches that are uncertain to happen (for example, losers’ bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches. 
- If a match needs rescheduling to a time after a related losers’ bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed. 

## Links

![Links](img/links.png)

> - [Forum Post](https://osu.ppy.sh/community/forums/topics/940042)

Reviewer: ```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/940042)
```

> - [Discord](https://discord.gg/jYhjTtj)

Reviewer: ```suggestion
- [UKCC Discord server](https://discord.gg/jYhjTtj)
```

> - [Spreadsheet (Map pools, Schedules, Stats)](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)

Reviewer: ```suggestion
- [Statistics sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)
```
Put this on the very bottom of links section, and bold it.

> - [Twitch](https://twitch.tv/osuukcc)

Reviewer: ```suggestion
- [Livestream](https://twitch.tv/osuukcc)
```

> - [Bracket (Challonge)](https://challonge.com/ukcc7)

Reviewer: ```suggestion
- [Challonge brackets](https://challonge.com/ukcc7)
```

Author: Used `bracket` singular 👍

> - [Pick'em](https://pickem.hwchr.com/tournaments/15)

Reviewer: ```suggestion
- [Pick'em predictions website](https://pickem.hwchr.com/tournaments/15)
```

## Rules

![Rules](img/rules.png)

- Referees will create the lobby approximately **10 minutes** before the match is due to start. 
- Players have a 10 minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will recieve losses.
- We take no-shows seriously, if you fail to show up to your match without valid reason, you may forfeit your right to future UKCC tournaments.
- The referee cannot play during the match.

### __**NEW**: Qualifying Rounds__

> - The first two rounds will be qualifying rounds, where each player will play a total of 7 maps

Reviewer: ```suggestion
- The first two rounds will be qualifying rounds, where each player will play a total of 7 maps

```
Add Space

- There will be 4 No Mod, 1 Hidden, 1 Hardrock and 1 Double Time map in each round
- Your scores on each map will be added together giving you a total score
- The top 64 players will progress from the first qualifier
- The top 32 players will progress from the second qualifier
- Seeding for groups will be determined from your seed based on your osu! rank (as of signup close) and your overall position in the RO64 qualifer, eg 3rd seed + 15th in qualifier = 18 points

> ### Rolling

Reviewer: ```suggestion
### Rolling

```
Add space

- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](https://i.imgur.com/S0VIvni.png).

> ### Banning

Reviewer: ```suggestion

### Banning

```
Add whitespace

- From the Group Stage onwards players will have **TWO** bans.
- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.

> ### Warmups

Reviewer: ```suggestion

### Warmups

```
```suggestion

### Banning
```
Add whitespace

- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less

> ### Picking

Reviewer: ```suggestion

### Picking

```
```suggestion

### Banning
```
Add whitespace

- Players have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
- For Freemod picks, players must select any combination of the following mods: **Hidden, HardRock**
- **NEW** - Failed scores WILL count
- Tiebreakers are Freemod, however No Mod can be used.

> ## Tournament Schedule

Reviewer: ```suggestion
## Tournament schedule
```
Sentence case

| Round of 32 | 2019-09-21/2019-09-22 |
| Round of 16 + Losers Round 1 | 2019-09-28/2019-09-29 |
| Quarter Finals + Losers Round 2/3 | 2019-10-5/2019-10-6 |
| Semi Finals + Losers Round 4/5 | 2019-10-12/2019-10-13 |
| Winners Finals + Losers 6/Semi Final | 2019-10-19/2019-10-20 |
| Losers Final + GRAND Finals | 2019-10-26/2019-10-27 |

## Prizes

![Prizes](img/prizes.png)

| Placing | Prize(s)|
| :-: | :-- |
| ![Gold Crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |
| ![Silver Crown](/wiki/shared/crown-silver.png "Second place") | 3 months of osu! supporter and a sparkling medal |
| ![Bronze Crown](/wiki/shared/crown-bronze.png "Third place") | 1 month of osu supporter and a sparkling medal |

### Badge
![UKCC 7 Winner](img/badge.png)

## Organisation

![Staff List](img/staff.png)

UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [Discord](https://discord.gg/jYhjTtj)!

| Position | Member(s) |
| :-- | :-- |

> | Host(s) | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |

Reviewer: ```suggestion
| Host | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
```
or preferably Manager

Author: There's multiple hosts - not sure I agree with this one 🤔

> | Referees | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_DE] [Bakugo-](https://osu.ppy.sh/users/4990127), ![][flag_GB] [Domblade](https://osu.ppy.sh/users/6701945), ![][flag_BE] [DraQeu](https://osu.ppy.sh/users/4425520), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_FR] [Mimir](https://osu.ppy.sh/users/7382734), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264) |

Reviewer: ```suggestion
| Referee | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_DE] [Bakugo-](https://osu.ppy.sh/users/4990127), ![][flag_GB] [Domblade](https://osu.ppy.sh/users/6701945), ![][flag_BE] [DraQeu](https://osu.ppy.sh/users/4425520), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_FR] [Mimir](https://osu.ppy.sh/users/7382734), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264) |
```

> | Commentators | ![][flag_GB] [sut_son](https://osu.ppy.sh/users/9622163), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_GB] [Doomsday is bad](https://osu.ppy.sh/users/3481378), ![][flag_GB] [Daiyaa](https://osu.ppy.sh/users/929134), + a lot of the UK community |

Reviewer: ```suggestion
| Commentator | ![][flag_GB] [sut_son](https://osu.ppy.sh/users/9622163), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_GB] [Doomsday is bad](https://osu.ppy.sh/users/3481378), ![][flag_GB] [Daiyaa](https://osu.ppy.sh/users/929134), + a lot of the UK community |
```

> | Streamers | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850) |

Reviewer: ```suggestion
| Streamer | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [_Takara](https://osu.ppy.sh/users/11599184), ![][flag_GB] [Kingling](https://osu.ppy.sh/users/7010761), ![][flag_GB] [xFluffyo](https://osu.ppy.sh/users/8397226), ![][flag_FI] [Lefafel](https://osu.ppy.sh/users/2295850) |
```

> | Mappoolers | ![][flag_BR] [Dada](https://osu.ppy.sh/users/9119507), ![][flag_US] [\_p0ke\_](https://osu.ppy.sh/users/929134) |

Reviewer: ```suggestion
| Mappooler | ![][flag_BR] [Dada](https://osu.ppy.sh/users/9119507), ![][flag_US] [\_p0ke\_](https://osu.ppy.sh/users/929134) |
```
Consider Map selector?

> | Statistics | ![][flag_NO] [Yokespai](https://osu.ppy.sh/users/6399568) |

Reviewer: ```suggestion
| Statistician | ![][flag_NO] [Yokespai](https://osu.ppy.sh/users/6399568) |
```

> | Graphics | ![][flag_GB] [Melonturtle](https://osu.ppy.sh/users/4775283), ![][flag_GB] [Listie](https://osu.ppy.sh/users/4121535) |

Reviewer: ```suggestion
| Graphics | ![][flag_GB] [Melonturtle](https://osu.ppy.sh/users/4775283), ![][flag_GB] [Listie](https://osu.ppy.sh/users/4121535) |
```
Designer, maybe?

Reviewer 2: yeah let's go with "Designer" or "Graphics designer", it should be a role here

> 

Reviewer: Remove these whitespaces, you only need 1 line break.

> ## Players

Reviewer: ```suggestion
## Players

```
Line break

| 97-111 | ![][flag_GB] [Damarsh](https://osu.ppy.sh/users/7465147), ![][flag_GB] [DrexiN](https://osu.ppy.sh/users/5363646), ![][flag_GB] [cfman29](https://osu.ppy.sh/users/9023597), ![][flag_GB] [God](https://osu.ppy.sh/users/10502908), ![][flag_GB] [Mantle](https://osu.ppy.sh/users/7225966), ![][flag_GB] [aroS](https://osu.ppy.sh/users/7193474), ![][flag_IE] [Neverslain](https://osu.ppy.sh/users/10644912), ![][flag_IE] [kaniciwah](https://osu.ppy.sh/users/9371954), ![][flag_GB] [josph12](https://osu.ppy.sh/users/8584840), ![][flag_GB] [mamoopy](https://osu.ppy.sh/users/9384951), ![][flag_GB] [fadentic](https://osu.ppy.sh/users/13453905), ![][flag_GB] [Metzo](https://osu.ppy.sh/users/10633982), ![][flag_GB] [Bowris](https://osu.ppy.sh/users/8562850), ![][flag_GB] [_Phazy](https://osu.ppy.sh/users/3956707), ![][flag_IE] [Broznix](https://osu.ppy.sh/users/13909259) |

> ---

Reviewer: ```suggestion

---

```
Line break

> ## RESULTS

Reviewer: ```suggestion
## Podium
```
Fits better plus no caps please

UKCC 7 concluded on the 27th October 2019.

> | Placing | Player|

Reviewer: ```suggestion
| Placing | Player |
```

| :-: | :-- |

> | ![Gold Crown](/wiki/shared/crown-gold.png "Winner") | [Bubbleman](https://osu.ppy.sh/users/5182050) |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | [Bubbleman](https://osu.ppy.sh/users/5182050) |
```
Sentence case, apply for all

| ![Silver Crown](/wiki/shared/crown-silver.png "Second place") | [Spare](https://osu.ppy.sh/users/2204373) |
| ![Bronze Crown](/wiki/shared/crown-bronze.png "Third place") | [AJT](https://osu.ppy.sh/users/3181083) |

---

## Map pools

![Map pools](img/mappool.png)

*NB - Map packs may no longer be available due to age.*

### Losers Final + Grand Final

**[Download the map pack here! (148.0MB)](https://mega.nz/#!TmhgUK5K!7kkYZPdolGvJbuCezRBj1SFJe6J-qZRu0B4qXTwXwzU)**

- Nomod
    - [Shiena Nishizawa - Brand-new World [Brand-new Extra]](https://osu.ppy.sh/b/1391381)
    - [xi - over the top [FOUR DIMENSIONS]](https://osu.ppy.sh/b/1682412)
    - [rerulili - Noushou Sakuretsu Girl [sanji no oyatsu wa makaron tabemasu]](https://osu.ppy.sh/b/911946)
    - [lapix - Amazing Mirage (Extended) [Fata Morgana]](https://osu.ppy.sh/b/1232752)
    - [Umeboshi Chazuke - Bison Charge [Flask's Extreme]](https://osu.ppy.sh/b/1900116)

>     - [sakuzyo - AXION [AXION_REBORN]](https://osu.ppy.sh/b/324516)

Reviewer: ```suggestion
    - [sakuzyo - AXION [AXION\_REBORN]](https://osu.ppy.sh/b/324516)
```
Escape

    - [Yunomi & nicamoq - Indoor Kei Nara Trackmaker [Another]](https://osu.ppy.sh/b/1115999)
    - [DJKurara - Goodbye [SEVEN COLORS]](https://osu.ppy.sh/b/1811477)
- Hard Rock
    - [TAG - PRANA [Extreme]](https://osu.ppy.sh/b/234444)
    - [Yumeno Yuki - THE APPLE IS CAST! [Resolve]](https://osu.ppy.sh/b/1974604)
    - [Powerless - Soleil [#1: handsome & pishifat]](https://osu.ppy.sh/b/1523514)
- Double Time
    - [Toyosaki Aki - MORE&MORE [Insane]](https://osu.ppy.sh/b/318975)
    - [BRZion - Game Brain [Insane]](https://osu.ppy.sh/b/105978)
    - [kors k - Dot [korzhik]](https://osu.ppy.sh/b/441616)
    - [Hardgore Alice (CV: Hidaka Rina) - Forget me Not... [Insane]](https://osu.ppy.sh/b/1924965)
- Free Mod
    - [DECO*27 - Ghost Rule [z1085684963's Extra]](https://osu.ppy.sh/b/951521)
    - [DragonForce - Ashes of the Dawn [Extra]](https://osu.ppy.sh/b/1650328)
    - [CustomiZ - COOLEST [Hi-Speed Extreme]](https://osu.ppy.sh/b/1105536)
- Tiebreaker
    - [Ata - Euphoria [Delusion]](https://osu.ppy.sh/b/1705819)

### Winners Final + Losers Round 6/SF

**[Download the map pack here! (165.6MB)](https://mega.nz/#!7ux0kKbD!aBqtZjDCQnxb4nvianpVv_2eOUl4aEeMa6TmZH-w5so)**

- Nomod

>     - [*namirin - Amatsu Kitsune [Extra]](https://osu.ppy.sh/b/881313)

Reviewer: ```suggestion
    - [\*namirin - Amatsu Kitsune [Extra]](https://osu.ppy.sh/b/881313)
```
Oopsie, missed an escape

    - [Nekomata L.E.D.Master+ - Chrono Diver -PENDULUMs- [Theaceae's Another]](https://osu.ppy.sh/b/1628965)
    - [siromaru + cranky - conflict [Encounter]](https://osu.ppy.sh/b/1955676)
    - [t+pazolite - Call me it. (500 Tortures) [Extra]](https://osu.ppy.sh/b/655682)
    - [Camellia - Chirality [fanzhen's Extra]](https://osu.ppy.sh/b/1464581)

>     - [Pastel*Palettes - Happy Synthesizer [Shizuku's Expert]](https://osu.ppy.sh/b/1989990)

Reviewer: ```suggestion
    - [Pastel\*Palettes - Happy Synthesizer [Shizuku's Expert]](https://osu.ppy.sh/b/1989990)
```
Escape

- Hidden
    - [In Flames - Take This Life [kiry's Extra]](https://osu.ppy.sh/b/1569878)
    - [Tachibana Miya - Miya to Tengoku to Jigoku [Hentai]](https://osu.ppy.sh/b/62269)
    - [Camellia as "Reverse of Riot" - Completeness Under Incompleteness [RLC's Infinite]](https://osu.ppy.sh/b/1094551)
- Hard Rock
    - [Hana - Sakura no Uta [VI.Artist of the Sakura]](https://osu.ppy.sh/b/994933)
    - [Glamour of the Kill - A Hope in Hell [Hopeless]](https://osu.ppy.sh/b/104389)
    - [Toby Fox - Chaos King [Kalibe's Expert]](https://osu.ppy.sh/b/1884623)
- Double Time
    - [NanosizeMir - Philosophyz -NanosizeMir Ver.- [Insane]](https://osu.ppy.sh/b/1351933)
    - [TK from Ling tosite sigure - White Out [Snowy Departure]](https://osu.ppy.sh/b/1418668)
    - [Amane - Midsummer Festival [Lunatekke]](https://osu.ppy.sh/b/75822)
    - [NEWS - weeeek [Insane]](https://osu.ppy.sh/b/22422)
- Free Mod
    - [Byakko - Kachou Fuusetsu [MasterPeeysa]](https://osu.ppy.sh/b/453790)
    - [Denkishiki Karen Ongaku Shuudan - Owari Tsumugishi Mono [Lasse's Extra]](https://osu.ppy.sh/b/1711326)

>     - [Kucchy vs Akky - Satori ~3rd eyes~ [Lunatic]](https://osu.ppy.sh/b/50892)

Reviewer: ```suggestion
    - [Kucchy vs Akky - Satori \~3rd eyes\~ [Lunatic]](https://osu.ppy.sh/b/50892)
```
Escape

- Tiebreaker

>     - [Demetori - Eastern Dream ~ The Introduction / Kamisabita Kosenjou ~ Suwa Foughten Field [Stage 6]](https://osu.ppy.sh/b/1362265)

Reviewer: ```suggestion
    - [Demetori - Eastern Dream ~ The Introduction / Kamisabita Kosenjou \~ Suwa Foughten Field [Stage 6]](https://osu.ppy.sh/b/1362265)
```
Escape

### Semi Final + Losers Round 4/5

**[Download the map pack here! (128.7MB)](https://mega.nz/#!S7p2QShR!ugelC74o3C1Xc3ECMHCb7EWzhvuZh75UuGs9_nznR44)**

- Nomod
    - [Roselia - R [Rosenlied]](https://osu.ppy.sh/b/1719416)
    - [LeaF - MEPHISTO [Mephistopheles]](https://osu.ppy.sh/b/1509096)
    - [Reol - Yakusoku no Ao [Chaoz's Extra]](https://osu.ppy.sh/b/1011937)
    - [Camellia feat. Nanahira - NANI THE FUCK!! [???????]](https://osu.ppy.sh/b/2137778)
    - [EZFG - Hurting for a Very Hurtful Pain [Dance]](https://osu.ppy.sh/b/145669)
    - [kamome sano - citrus [moecho's lemonade]](https://osu.ppy.sh/b/1411721)
- Hidden
    - [Nico Nico Chorus - Unhappy Refrain [Trigger-Happy]](https://osu.ppy.sh/b/1129480)
    - [RevenG - Sakura [Insane]](https://osu.ppy.sh/b/25296)
    - [Asaka - Open your eyes [World: :Synchronization]](https://osu.ppy.sh/b/1871739)
- Hard Rock
    - [Maksim Mrvica - Croatian Rhapsody [Presto]](https://osu.ppy.sh/b/1778453)

>     - [Ryu* - Sakura Reflection (MYTK Remix) [Another]](https://osu.ppy.sh/b/650610)

Reviewer: ```suggestion
    - [Ryu\* - Sakura Reflection (MYTK Remix) [Another]](https://osu.ppy.sh/b/650610)
```
Escape

    - [Azel - -Astraia- [Extra]](https://osu.ppy.sh/b/706743)
- Double Time
    - [TRUE - DREAM SOLISTER [Insane]](https://osu.ppy.sh/b/2026140)
    - [Hiro - Carol Megamix [Insane]](https://osu.ppy.sh/b/190584)
    - [Two Door Cinema Club - Cigarettes In The Theatre [Insane]](https://osu.ppy.sh/b/120461)
    - [zts - worldenddominator [Dada's Insane]](https://osu.ppy.sh/b/1985639)
- Free Mod
    - [All That Remains - This Calling [Despair]](https://osu.ppy.sh/b/682053)
    - [Wiklund - Billy Boogie [16 Bits?!]](https://osu.ppy.sh/b/36547)
    - [Akiyama Uni - The Grimoire of Alice [Extra]](https://osu.ppy.sh/b/316018)
- Tiebreaker
    - [USAO - Showdown [Calamity]](https://osu.ppy.sh/b/1456429)

### Quarter Final + Losers Round 2/3

**[Download the map pack here! (141.7MB)](https://mega.nz/#!37BHiSQa!_YCHsfzKe3AqxPboSBa2Uuyf0_chbWqYVTpd5QNj9WA)**

- Nomod
    - [Suzumu - Kakumeisei ousama densenbyou [Extra]](https://osu.ppy.sh/b/688304)

>     - [Prim - Koisuru*Uchuu Sensou!! [Another]](https://osu.ppy.sh/b/231964)

Reviewer: ```suggestion
    - [Prim - Koisuru\*Uchuu Sensou!! [Another]](https://osu.ppy.sh/b/231964)
```
Escape

    - [flumpool - Yoru wa Nemureru kai? [Insomnia]](https://osu.ppy.sh/b/1557509)
    - [Camellia - crystallized [Ca2Al3(SiO4)3(OH)]](https://osu.ppy.sh/b/867819)
    - [Imperial Circus Dead Decadence - Shinbatsu o Tadori Kyoukotsu ni Itaru [Extra Collab]](https://osu.ppy.sh/b/2040764)
    - [Shindig - New Friends [squirrelp's Key to a Happy Life]](https://osu.ppy.sh/b/1603103)
- Hidden
    - [Hiroyuki Sawano feat. Mika Kobayashi - Bios [LKs' Another]](https://osu.ppy.sh/b/137766)
    - [DIGI-SEQ-BAND2000 - Somehow You Found Me [Expert]](https://osu.ppy.sh/b/44692)

>     - [Ryu* vs. Sota - Go Beyond!! [Nuvolina's Expert!!]](https://osu.ppy.sh/b/2023961)

Reviewer: ```suggestion
    - [Ryu\* vs. Sota - Go Beyond!! [Nuvolina's Expert!!]](https://osu.ppy.sh/b/2023961)
```
Escape

    - [JYOCHO - Taiyou to Kurashite Kita [Bloom]](https://osu.ppy.sh/b/1269564)
- Hidden
    - [S3RL - MTC (Different Heaven Remix) [Heaven]](https://osu.ppy.sh/b/770354)
    - [Conagusuri - Hina Choco Dark Matter [Lunatic]](https://osu.ppy.sh/b/56337)
    - [adaptor - LAISSEZ:FAIRE [Team Sample Text's Extra]](https://osu.ppy.sh/b/1419707)
- Hard Rock
    - [Giga & Suzumu feat. Kagamine Rin - Undead Enemy [Oppression]](https://osu.ppy.sh/b/524187)
    - [Qrispy Joybox feat.mao - Umeyukiyo [Insane]](https://osu.ppy.sh/b/265643)
    - [Levaslater - NNRT [Extra]](https://osu.ppy.sh/b/146985)
- Double Time
    - [Reol - MONSTER [SHARKIE'S INSANE]](https://osu.ppy.sh/b/812721)
    - [VY1 - Cyber Thunder Cider [Cyber]](https://osu.ppy.sh/b/151215)
    - [David Hasselhoff - True Survivor [Insane Survivors]](https://osu.ppy.sh/b/737878)
- Free Mod
    - [Cranky - The Five [Another]](https://osu.ppy.sh/b/64996)
    - [Airi Totoki, Akane Hino, Aiko Takamori, Syoko Hoshi, Yuko Hori - Zettai Tokken Shuchou Shimasu! [MASTER]](https://osu.ppy.sh/b/1559184)
    - [Rigel Theatre - Rinn Ding Dong [moph's Big Dong Extra]](https://osu.ppy.sh/b/1724396)
- Tiebreaker

>     - [Demetori - Youyoumu ~ The Woman Who Sold the Moon [Phantasm]](https://osu.ppy.sh/b/1562486)

Reviewer: ```suggestion
    - [Demetori - Youyoumu \~ The Woman Who Sold the Moon [Phantasm]](https://osu.ppy.sh/b/1562486)
```
Escape

### RO32

**[Download the map pack here! (114.6MB)](https://mega.nz/#!OvQhmAiS!PT1qpuu5XGjnvX9Ra9w8flaEZsL_CCW47scekuARt7Y)**

- Nomod
    - [kalon. - Juubaku [rui's Extra]](https://osu.ppy.sh/b/1123161)
    - [Yuuyu - Scrap Syndrome [Extra]](https://osu.ppy.sh/b/2044722)
    - [Eagle - Hypersonik [NeilPerry]](https://osu.ppy.sh/b/1104818)
    - [TK from Ling Tosite Sigure - katharsis [Expert]](https://osu.ppy.sh/b/1803966)
    - [MAX MAXIMIZER vs. DJ TOTTO - Rebellio [REALARU's EXTRA]](https://osu.ppy.sh/b/1632337)
    - [Meg & Dia - Monster (DotEXE Remix) [Xtra]](https://osu.ppy.sh/b/436825)
- Hidden
    - [Kuroneko Dungeon - Lilieze to Enryuu Laevateinn [Another†leggendaria]](https://osu.ppy.sh/b/725139)

>     - [ZUN - The Youkai Mountain ~ Mysterious Mountain [Lunatic]](https://osu.ppy.sh/b/76917)

Reviewer: ```suggestion
    - [ZUN - The Youkai Mountain \~ Mysterious Mountain [Lunatic]](https://osu.ppy.sh/b/76917)
```
Escape

>     - [Another Infinity feat. Mayumi Morinaga - COME BACK TO MY HEART (Ryu* Remix) [Extreme]](https://osu.ppy.sh/b/186599)

Reviewer: ```suggestion
    - [Another Infinity feat. Mayumi Morinaga - COME BACK TO MY HEART (Ryu\* Remix) [Extreme]](https://osu.ppy.sh/b/186599)
```
Escape

- Hard Rock
    - [Breaking Benjamin - The Diary of Jane [Extreme]](https://osu.ppy.sh/b/1001921)
    - [Eagle - S!ck [Extra]](https://osu.ppy.sh/b/341285)
    - [An - Encryption [Kloyd's Extra]](https://osu.ppy.sh/b/258384)
- Double Time

>     - [Nanamori-chu * Goraku-bu - Precious Friends [Insane]](https://osu.ppy.sh/b/420131)

Reviewer: ```suggestion
    - [Nanamori-chu \* Goraku-bu - Precious Friends [Insane]](https://osu.ppy.sh/b/420131)
```
Escape

---
tags:

>   - UK Community Cup 7

Reviewer 2: the title doesn't need to be repeated in the tags (every part of the title basically acts like a tag)
```suggestion
```

>   - UKCC 7

Reviewer 2: likewise no *word* of the title needs to be repeated in the tags if it's not restricting the tag on purpose. here, the "7" is already matched by the title, and I'd want to be able to find this article by searching "UKCC" anyway
```suggestion
  - UKCC
```

>   - UKCC 7

Reviewer 2: ```suggestion
```
was written twice.

---

# UK Community Cup 7

![UKCC 7 logo](img/banner.png)

> The seventh edition of the bi-annual UK Community Cup, a tournament for those who live or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th October 2019.

Reviewer 2: generally all wiki articles introduce the main topic as the subject of the first sentence, and bold it too. something like this would be better
```suggestion
The **UK Community Cup 7** (***UKCC 7***) is the seventh edition of the bi-annual UK Community Cup, a tournament series for those who live in or are from the UK or Ireland. This edition was a 1v1 and finished on the 27th of October, 2019.
```
maybe add flag icons before the countries too, idk

Reviewer 2: also I didn't realise this tournament was already over. so it should be written in past tense (this `was` the seventh edition of ...)

> ---

Reviewer 2: don't need any of these ---

## Introduction

![Introduction](img/intro.png)

> - This is a **1v1, UK and Ireland based tournament**

Reviewer 2: this feels more like a list of rules rather than an introduction. probably move to that section, and if you want any more of the important info here still you should use sentences/paragraphs instead of bullet points

- If it is impossible to find a time to reschedule to that suits both players, both players should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
- Matches that are uncertain to happen (for example, losers’ bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches.
- If a match needs rescheduling to a time after a related losers’ bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed.

## Links

![Links](img/links.png)

- [Discussion Thread](https://osu.ppy.sh/community/forums/topics/940042)
- [UKCC Discord server](https://discord.gg/EtsU7Cw)
- [Livestream](https://twitch.tv/osuukcc)
- [Challonge bracket](https://challonge.com/ukcc7)
- [Pick'em predictions website](https://pickem.hwchr.com/tournaments/15)
- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)**

> ## Rules

Reviewer 2: rules are usually put as the last section in tournament articles since they get pretty long and the only people who would rly care about this are the participants

Author: The images we made for our forum post are numbered as you can see:

![rules](https://user-images.githubusercontent.com/31848108/76759264-3b361c00-6783-11ea-8b72-fc9a40c12a82.png)

Would it be okay to overlook this for this reason?

Reviewer 2: is it not possible to renumber them here

Author: I didn't make the graphics 😢 so not really. It's not the end of the world if it's out of order.

![Rules](img/rules.png)

- Referees will create the lobby approximately **10 minutes** before the match is due to start.
- Players have a 10 minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will recieve losses.
- We take no-shows seriously, if you fail to show up to your match without valid reason, you may forfeit your right to future UKCC tournaments.
- The referee cannot play during the match.

> ### __**NEW**: Qualifying Rounds__

Reviewer 2: ```suggestion
### Qualifying rounds
```
assuming "new" is because this wasn't in ukcc6, you could mention that in a sentence below this header

- The first two rounds will be qualifying rounds, where each player will play a total of 7 maps
- There will be 4 No Mod, 1 Hidden, 1 Hardrock and 1 Double Time map in each round
- Your scores on each map will be added together giving you a total score
- The top 64 players will progress from the first qualifier
- The top 32 players will progress from the second qualifier
- Seeding for groups will be determined from your seed based on your osu! rank (as of signup close) and your overall position in the RO64 qualifer, eg 3rd seed + 15th in qualifier = 18 points

### Rolling

> - When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](https://i.imgur.com/S0VIvni.png).

Reviewer 2: ```suggestion
- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
```
additionally you might want to move this image to the wiki so it's guaranteed to stay working

Author: How do I format this inline if i'm adding the png to a file? Is it like this and will the link work?
```suggestion
- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](img/ukcc4rollsystem.png).
```

Reviewer 2: yep, you can test it in the github preview too

### Banning

- From the Group Stage onwards players will have **TWO** bans.
- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.

### Warmups

- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less

### Picking

- Players have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
- For Freemod picks, players must select any combination of the following mods: **Hidden, HardRock**
- **NEW** - Failed scores WILL count
- Tiebreakers are Freemod, however No Mod can be used.

## Tournament schedule

![Schedules](img/schedules.png)

| Event | Timestamp |
| --: | :-- |

> | Signups Open | 2019-07-27 |

Reviewer 2: in general, sentence case things that aren't titles. for this first one it'd be "Signups open"

> | Signups Closed | 2019-08-26 |

Reviewer 2: closed => close

same tense for all of these

| Qualifying Round 1 | 2019-09-07/2019-09-08 |
| Qualifying Round 2 | 2019-09-14/2019-09-15 |
| Round of 32 | 2019-09-21/2019-09-22 |
| Round of 16 + Losers Round 1 | 2019-09-28/2019-09-29 |
| Quarter Finals + Losers Round 2/3 | 2019-10-5/2019-10-6 |
| Semi Finals + Losers Round 4/5 | 2019-10-12/2019-10-13 |
| Winners Finals + Losers 6/Semi Final | 2019-10-19/2019-10-20 |
| Losers Final + Grand Finals | 2019-10-26/2019-10-27 |

## Prizes

![Prizes](img/prizes.png)

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu!supporter tag, a shiny profile badge and a magnificent trophy! |
```
Product name

---

## Podium

UKCC 7 concluded on the 27th October 2019.

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | [Bubbleman](https://osu.ppy.sh/users/5182050) |
| ![Silver crown](/wiki/shared/crown-silver.png "Second place") | [Spare](https://osu.ppy.sh/users/2204373) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "Third place") | [AJT](https://osu.ppy.sh/users/3181083) |

---

## Map pools

![Map pools](img/mappool.png)

> *NB - Map packs may no longer be available due to age.*

Reviewer 2: ```suggestion
*Note: Map packs may no longer be available due to age.*
```
standard way to write notes on wiki

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Spare** ![][flag_GB] | **6** | 3 | ![][flag_GB] AJT | [#1](https://osu.ppy.sh/community/matches/55673675) |

Sunday, 20 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Harpy** ![][flag_GB] | **6** | 4 | ![][flag_GB] AJT| [#1](https://osu.ppy.sh/community/matches/55693071) |

Tuesday, 21 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Ventus ![][flag_GB] | 4 | **6** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55719453) |

Monday, 22 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Harpy ![][flag_GB] | 4 | **6** | ![][flag_GB] **Bubbleman** | [#1](https://osu.ppy.sh/community/matches/55738843) |

### Semi Final + Losers Round 4 / 5

Friday, 11 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| Malteser ![][flag_GB] | 2 | **6** | ![][flag_GB] **Best459** | [#1](https://osu.ppy.sh/community/matches/55482259) |

Saturday, 12 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |

> | bloo ![][flag_GB] | - | **1** | ![][flag_GB] **Kingling** | FF |

Reviewer: ```suggestion
| bloo ![][flag_GB] | -1 | **0** | ![][flag_GB] **Kingling** | *win by default* |
```
I prefer *win by default* (with italics) don't know if it's necessary, plz check clayton.
Scoring is just my example of how i usually takes it.

Reviewer 2: `*win by default*`, `*forfeit*`, anything along these lines is ok. i would avoid abbreviations by themselves cuz ppl might not be familiar with them

Author: Went with *forfeit* 👍 leaving the scoring as is, strongly opposed to `-1` as a score

Monday, 2019-9-30

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Apo11o** ![][flag_GB] | **5** | 2 | ![][flag_GB] Defex | [#1](https://osu.ppy.sh/community/matches/55236437) |

Friday, 5 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Kingling** ![][flag_GB] | **1** | - | ![][flag_GB] meltz | FF |
| **Spare** ![][flag_GB] | **5** | 0 | ![][flag_GB] bloo | [#1](https://osu.ppy.sh/community/matches/55335898) |
| MoeYandere ![][flag_GB] | - | **1** | ![][flag_GB] hlo | FF |
| **Kingling** ![][flag_GB] | **1** | - | ![][flag_GB] Apo11o | FF |
| majoreh ![][flag_GB] | 2 | **5** | ![][flag_GB] Zyphox | [#1](https://osu.ppy.sh/community/matches/55347021) |
| **AJT** ![][flag_GB] | **5** | 3 | ![][flag_GB] Best459 | [#1](https://osu.ppy.sh/community/matches/55350496) |

Sunday, 6 October 2019

| Player 1 | | | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Malteser** ![][flag_GB] | **5** | 3 | ![][flag_GB] Woffel | [#1](https://osu.ppy.sh/community/matches/55368204) |
| **hlo** ![][flag_GB] | **5** | 3 | ![][flag_GB] Zyphox | [#1](https://osu.ppy.sh/community/matches/55368217) |

> | **Bubbleman** ![][flag_GB] | **5** | 1 | ![][flag_GB] Rinnu | [#1](https://osu.ppy.sh/mp/55371230) |

Reviewer: ```suggestion
| **Bubbleman** ![][flag_GB] | **5** | 1 | ![][flag_GB] Rinnu | [#1](https://osu.ppy.sh/community/matches/55371230) |
```
